src/App.tsx

```diff
@@ -13,52 +13,31 @@ interface FishkappCard {
 }
 
 interface iCard{
-  id: number,
-  question: string,
-  answer: string,
+  _id: number,
+  face: string,
+  back: string,
 }
 
 function App() {
+  
+
   const [displayNewCard, setDisplayNewCard] = useState<boolean>(false)
   const [editMode, setEditMode] = useState<boolean>(false);
-  const [cardsData, setCardsData] = useState<iCard[]>([
-    {
-      id: 1,
-      question: "1:How do you write comment in JS?",
-      answer:
-        "its //comment",
-    },
-    {
-      id: 2,
-      question: "How do you write comment in JS?",
-      answer:
-        "its //comment",
-    },
-    {
-      id: 3,
-      question: "How do you write comment in JS?",
-      answer:
-        "its //comment",
-    },
-    {
-      id: 4,
-      question: "How do you write comment in JS?",
-      answer:
-        "its //comment",
-    }
-  ]);
+  const [cardsData, setCardsData] = useState<iCard[]>([]);
 
   
 
   const addCard = () => {
     setDisplayNewCard(true);
   };
-    
 
   const saveCard = async ( card: FishkappCard) => {
+    setDisplayNewCard(false);
     try {
       const res = await addCardService(card.question, card.answer);
+      await addCardService(card.question, card.answer).then(data => console.log(data));
```

Comment:
**issue:** Looks like `addCardService` is called twice here

Reply:
Issue fixed